src/opendr/perception/pose_estimation/hr_pose_estimation/high_resolution_learner.py

```diff
@@ -681,8 +1215,377 @@ def infer(self, img, upsample_ratio=4, stride=8, track=True, smooth=True, multis
             else:
                 heatmap = self.prev_heatmap
         self.counter += 1
+        bounds = ([self.xmin, self.xmax, self.ymin, self.ymax],)
+        return current_poses, heatmap, bounds
+
+    def infer_adaptive(self, img, upsample_ratio=4, stride=8, track=True, smooth=True, multiscale=False):
```

Comment:
Arguments not actually used.
```suggestion
    def infer_adaptive(self, img, upsample_ratio=4, stride=8):
```